.circleci/config.yml

```diff
@@ -32,6 +32,15 @@ commands:
       - run:
           name: Run tests
           command: cargo test --workspace --all-features --jobs=3 <<#parameters.release>>--release<</parameters.release>>
+      - run:
```

Comment:
I don't think the `cargo build` step is necessary - `cargo test` already does this.